File: Data Structures & Algorithms/anagram-groups/submission-1.java
class Solution {
    public List<List<String>> groupAnagrams(String[] strs) {
        Map<String, List<String>> dict = new HashMap<>();
        for (int i = 0; i < strs.length; i++) {
            char[] sorted = strs[i].toCharArray();
            Arrays.sort(sorted);
            String ss = new String (sorted);
            if (dict.get(ss) == null) {
                dict.put(ss, new ArrayList<>());
            }
            dict.get(ss).add(strs[i]);
        }
        List<List<String>> res = new ArrayList<>(dict.values());
        return res;
    }
}

File: Data Structures & Algorithms/longest-common-prefix/submission-1.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        int longest = 0;
        for (int i = 0; i < strs[0].Length; i++) {
            for (int j = 0; j < strs.Length; j++) {
                if (strs[j].Length == 0) {
                    return "";
                }
                if (i >= strs[j].Length || strs[0][i] != strs[j][i]) {
                    return strs[0].Substring(0, i);
                }
            }
            longest++;
        }
        return strs[0].Substring(0, longest);
    }
}